openhtf/util/__init__.py

```diff
@@ -106,6 +108,23 @@ def __get__(self, instance, owner):
     return self._func(owner)
 
 
+def safe_format(target, **kwargs):
```

Comment:
Minor nit about naming here. To me the term "safe" in the context of string formatting hints towards the escaping of special characters (such as in preventing attacks like SQL injection). I'm not sure I have a great alternative here (maybe `partial_format`), but also don't want users to see this and get the wrong idea. Even if it would technically be their own fault for using code they don't understand.